### PR TITLE
Escape hatches

### PR DESCRIPTION
**Pull request by fraincs**
Issue:
Stylizing the doc page could be problematic when there is parts of the DOM without classes(escape hatches). This is true for the Preview component as well as the Props Table header and body.

## What I did

Added a class ``` sbdocs-preview ``` to the doc preview component

Added a class of  ``` docblock-propstable-head ``` and ``` docblock-propstable-body ``` to the props table doc component

## How to test

- Is this testable with Jest or Chromatic screenshots? No
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

### CODE REVIEW

File: lib/components/src/blocks/Preview.tsx [hunk 1]
```diff
@@ -154,7 +156,11 @@ const Preview: FunctionComponent<PreviewProps> = ({
   }
   const showToolbar = withToolbar && !Array.isArray(children);
   return (
-    <PreviewContainer {...{ withSource, withToolbar: showToolbar }} {...props}>
+    <PreviewContainer
+      {...{ withSource, withToolbar: showToolbar }}
+      {...props}
+      className="sbdocs-preview"
```
Comment (ndelangen) [inline lib/components/src/blocks/Preview.tsx]: I think this should accept a className from props and merge with `"sbdocs-preview"` if present?

**Review by ndelangen — CHANGES_REQUESTED** [review]

**Comment by fraincs** [comment]
@ndelangen done

**Review by ndelangen — APPROVED** [review]

File: lib/components/src/blocks/PropsTable/PropJsDoc.tsx [hunk 2]
```diff
@@ -7,24 +7,107 @@ interface PropJsDocProps {
   tags: JsDocTags;
 }
 
-const TBody = styled.tbody({ boxShadow: 'none !important' });
+export const Table = styled.table<{}>(({ theme }) => ({
```
Comment (domyen) [inline lib/components/src/blocks/PropsTable/PropJsDoc.tsx]: I think this is now outdated – can you merge `next` back into the branch to get the latest.

**Review by domyen — CHANGES_REQUESTED** [review]

**Comment by domyen** [comment]
The classname stuff looks good to me. But the other changes have since been updated on `next` so will hold off commenting on that till you merge `next` back into this branch. Let me know and I'll review again.

File: lib/components/src/blocks/PropsTable/PropJsDoc.tsx (hunk 2 above)
Comment (fraincs) [reply lib/components/src/blocks/PropsTable/PropJsDoc.tsx]: Perfect

**Review by domyen — APPROVED** [review]

**Pull request merged**